package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "voxell.speech",
```

Comment:
is this the requirements to be on openUPM? haha

Reply:
You can try it on your own. OpenUPM returns an error and said the package name is not acceptable. Because you should be an owner of the domain name. Take a look at how @asus4 resolved the problem. He just added `com.github` at the start if his repository name on github.

Reply:
i see haha thanks for the heads up :)